pages/index.js

```diff
@@ -43,60 +43,65 @@ export default function HomePage({ trips, setTrips }) {
 
   return (
     <>
-      <StyledImageWrapper>
-        <StyledImage
-          src="/travel-log.jpg"
-          height={600}
-          width={800}
-          layout="responsive"
-          alt="Travel Notebook"
-        />
-      </StyledImageWrapper>
-      <StyledHeading1>My Travel Log</StyledHeading1>
-      <StyledHeading2>Create a new Trip</StyledHeading2>
-      <TripInputForm handleAddTrip={handleAddTrip} />
-      <StyledHeading3>Upcoming Trips</StyledHeading3>
-      <StyledSection>
-        {upcomingTrips.length === 0 ? (
-          <p>You do not have any upcoming trips yet.</p>
-        ) : (
-          upcomingTrips.map((trip) => (
-            <>
-              <StyledLink href="/explore">
+      <Scrollbox>
+        <StyledImageWrapper>
+          <StyledImage
+            src="/travel-log.jpg"
+            height={600}
+            width={800}
+            layout="responsive"
+            alt="Travel Notebook"
+          />
+        </StyledImageWrapper>
+        <StyledLogo>Travel</StyledLogo>
+        <StyledHeading1>My Travel Log</StyledHeading1>
+        <StyledHeading2>Create a new Trip</StyledHeading2>
+        <TripInputForm handleAddTrip={handleAddTrip} />
+        <StyledHeading3>Upcoming Trips</StyledHeading3>
+        <section>
+          {upcomingTrips.length === 0 ? (
+            <p>You do not have any upcoming trips yet.</p>
+          ) : (
+            upcomingTrips.map((trip) => (
+              <>
```

Comment:
do you need the fragments here? Because you don't use them in line 78

Reply:
Oopsie :D